app/controllers/revisions_controller.rb

```diff
@@ -37,6 +40,11 @@ def recover
 
   private
 
+  def load_issues
```

Comment:
This controller is working too hard as part of #show to render a custom sidebar depending on the reckon.

First I thought let's move the sidebar rendering across Tylium to an Ajax fetch post render. However for most controllers, that'd be overkill (perhaps not!).

But in this case, where the sidebar is potentially completely different in each controller, I think it does make sense.

Let the #show view retrieve the revision, and render the #show partial and have a fetch that in a separate request loads the sidebar contents.